python/cuml/comm/serialize.py

```diff
@@ -30,6 +32,17 @@
     from distributed.protocol.cuda import cuda_deserialize, cuda_serialize
     from distributed.utils import log_errors
 
+    from distributed.protocol import register_generic
+
+    from cuml.dask.naive_bayes.naive_bayes import MultinomialNB
+
+    register_generic(MultinomialNB, "cuda",
+                     cuda_serialize, cuda_deserialize)
+    register_generic(MultinomialNB, "dask",
+                     dask_serialize, dask_deserialize)
```

Comment:
For backwards compatibility with older versions of Dask, would suggest the following change.

```suggestion
    try:
        # Requires Dask Distributed 2.12.0+
        register_generic(MultinomialNB, "cuda",
                         cuda_serialize, cuda_deserialize)
    except TypeError:
        pass
    register_generic(MultinomialNB)
```